keyberon/src/layout.rs

```diff
@@ -1638,6 +1693,14 @@ impl<'a, const C: usize, const R: usize, const L: usize, T: 'a + Copy + std::fmt
             .unwrap_or(self.default_layer)
     }
 
+    pub fn active_held_layers(&self) -> impl Iterator<Item = usize> + Clone + '_ {
```

Comment:
I think a clean way to make this a non-breaking change (or make it an explicitly opt-out breaking change) is to have a `bool` determining the behaviour of this function. One behaviour would be to include only the very last layer found, and the other behaviour would include all layers found.

A question would be what is the default? I think it probably makes sense to make it a breaking change by default, and allow users to opt-out if they like the old behaviour. I think the new behaviour is more preferable as the default moving forward.

Reply:
done in a43c1932f861f52fe21f8821ce054ab897e4bfff

Reply:
lmk if the names are alright, also maybe we want to do bool instead?

Reply:
I'm fine with all the internal code naming but I think the user-facing defcfg should use the full word. E.g. I would be fine with any of the following (the third example is a reference to the name of the action section in config.adoc).

```
    transparent-resolution
    transparency-resolution
    transparent-key-resolution
```

Reply:
Maybe it's better to have this option as boolean? There will ever be only possible 2 values anyway.

Reply:
Also if it were boolean, then we have a possibility to give it a name that indicates that it's an option enabling legacy behavior e.g. `enable-legacy-trans-behavior`. 

How do we even want to document new vs old behavior? Present the new one as an upgrate, or an alternative? Knowing this would help with naming of the defcfg option.

Reply:
Presenting as an alternative is preferable to me. I like the current configuration with explicit `layer-stack|to-base-layer` values, just needs a better key.

Reply:
Sure. 3cdd4ac6180ca524f34267f20e172b9207994b8d